diff --git a/Headers/Foundation/NSBackgroundActivityScheduler.h b/Headers/Foundation/NSBackgroundActivityScheduler.h
index f7b8d6ab0..c1a45d132 100644
--- a/Headers/Foundation/NSBackgroundActivityScheduler.h
+++ b/Headers/Foundation/NSBackgroundActivityScheduler.h
@@ -77,7 +77,7 @@ GS_EXPORT_CLASS
   NSActivityOptions _opts;
   id _token;
   NSString *_reason;
-  BLOCK_SCOPE GSScheduledBlock _block;
+  GSScheduledBlock _block;
 }
   
 - (instancetype) initWithIdentifier: (NSString *)identifier;
